diff --git a/formats/list.js b/formats/list.js
index f0ba12ec05a4322e140c1bd11490e8d2ae33ab46..31bf3045f4d59e5d79d30f59ddc4771bbd7ef3cc 100644
--- a/formats/list.js
+++ b/formats/list.js
@@ -30,8 +30,8 @@ class ListItem extends Block {
         e.preventDefault();
       }
     };
-    ui.addEventListener('mousedown', listEventHandler);
-    ui.addEventListener('touchstart', listEventHandler);
+    ui.addEventListener('click', listEventHandler);
+    ui.addEventListener('touchend', listEventHandler);
     this.attachUI(ui);
   }
   format(name, value) {
