diff --git a/node_modules/react-quill-new/node_modules/quill/formats/list.js b/node_modules/react-quill-new/node_modules/quill/formats/list.js
index f0ba12e..31bf304 100644
--- a/node_modules/react-quill-new/node_modules/quill/formats/list.js
+++ b/node_modules/react-quill-new/node_modules/quill/formats/list.js
@@ -30,8 +30,8 @@ class ListItem extends Block {
         e.preventDefault();
       }
     };
-    ui.addEventListener('mousedown', listEventHandler);
-    ui.addEventListener('touchstart', listEventHandler);
+    ui.addEventListener('click', listEventHandler);
+    ui.addEventListener('touchend', listEventHandler);
     this.attachUI(ui);
   }
   format(name, value) {
